org.gnome.Shotwell.json

```diff
@@ -45,8 +45,8 @@
             "sources" : [
                 {
                     "type" : "archive",
-                    "url" : "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.2.tar.xz",
-                    "sha256" : "9e035c4b755f46bfae70ba81cdcf8328b03f554373cec8c816e8b5680f85353c"
+                    "url" : "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.3.tar.xz",
```

Comment:
Already in the GNOME SDK

Reply:
Oh, nice. Sorry, I only saw your comment after clicking on merge :(